lib/beaker/hypervisor/aws_sdk.rb

```diff
@@ -464,16 +464,16 @@ def wait_for_status(status, instances, &block)
       end
     end
 
-    # Handles special checks needed for f5 platforms.
+    # Handles special checks needed for netdev platforms.
     #
-    # @note if any host is an f5 one, these checks will happen once across all
+    # @note if any host is an netdev one, these checks will happen once across all
     #   of the hosts, and then we'll exit
     #
     # @return [void]
     # @api private
-    def wait_for_status_f5()
+    def wait_for_status_netdev()
       @hosts.each do |host|
-        if host['platform'] =~ /f5/
+        if host['platform'] =~ /f5-|netscaler/
```

Comment:
No, cumulus works just fine.  These NetDev types actually use 'puppet device' and don't have puppet agents like Cumulus, Arista & Cisco